src/components/Event.vue

```diff
@@ -61,7 +61,8 @@
 
 <script>
 import moment from 'moment'
-import { L, LGeoJson } from 'vue2-leaflet'
+import L from 'leaflet'
```

Comment:
why did you have to make this change?

Reply:
I'm far from an expert on ES6 imports, but it seems that because`L` is the default export from `vue2-leaflet` it needs to be imported separately.

Reply:
Actually, it looks like I may also be able to do `import L, { LGeoJson } from `vue2-leaflet`

Reply:
Actually, scratch that, it's because `L` is coming from `leaflet` where `LGeoJson` is from the `vue2-leaflet`. The shared prefix is definitely a little confusing.